Sources/Extensions.swift

```diff
@@ -1,27 +1,48 @@
 //
-//  WHDate.swift
+//  Extensions.swift
 //  Wormholy-iOS
 //
-//  Created by Paolo Musolino on 18/04/18.
-//  Copyright © 2018 Wormholy. All rights reserved.
+//  Created by Sagar Dagdu on 4/27/20.
+//  Copyright © 2020 Wormholy. All rights reserved.
 //
 
-import UIKit
+import Foundation
 
-extension Date{
+///This file is a centralized place for all the extensions used in the framework.
+
+extension Bundle {
```

Comment:
Do you think it makes more readable to place all extensions into single file rather than to split extensions by class?